bigtable/docs/snippets_table.py

```diff
@@ -412,6 +412,88 @@ def test_bigtable_table_row():
     table.truncate(timeout=300)
 
 
+def test_bigtable_table_append_row():
+    # [START bigtable_table_append_row]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    row_keys = [b"row_key_1", b"row_key_2"]
+    row1_obj = table.append_row(row_keys[0])
+    row2_obj = table.append_row(row_keys[1])
+    # [END bigtable_table_append_row]
+
+    row1_obj.set_cell(COLUMN_FAMILY_ID, COL_NAME1, CELL_VAL1)
```

Comment:
This line [fails](https://source.cloud.google.com/results/invocations/4119c229-0ca1-4c2e-a19b-74ec4738bea9/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fbigtable/log) with:

```python
    def test_bigtable_table_append_row():
        # [START bigtable_table_append_row]
        from google.cloud.bigtable import Client
        client = Client(admin=True)
        instance = client.instance(INSTANCE_ID)
        table = instance.table(TABLE_ID)
        row_keys = [b"row_key_1", b"row_key_2"]
        row1_obj = table.append_row(row_keys[0])
        row2_obj = table.append_row(row_keys[1])
        # [END bigtable_table_append_row]
>       row1_obj.set_cell(COLUMN_FAMILY_ID, COL_NAME1, CELL_VAL1)
E       AttributeError: 'AppendRow' object has no attribute 'set_cell'
```

Reply:
It's fixed. I'm not very sure about these two calls:
https://github.com/googleapis/google-cloud-python/blob/fef10c08f71c362cb63c9e588d9d39f3dcd4a878/bigtable/noxfile.py#L174-L184

If first call will fail, the second one (for `snippets_table.py`) will be just skipped. It can be a little bit inconvenient: in some cases you'll see only first check fails, without second check results at all. It's better to see whole picture from the very beginning, I think. Still, I understand, why session written that way